packages/itwinui-css/backstop/tests/radio-tile.html

```diff
@@ -27,6 +27,15 @@
         display: flex;
         flex-direction: column;
         gap: 11px;
+        margin-bottom: 11px;
+      }
+      .right {
+        top: 0;
+        left: 100%;
+        margin-left: 5px;
+      }
+      .iui-input-container {
+        margin: 2px;
```

Comment:
This margin allows the test images to capture the outlines on the edges of the images. I think they aren't captured correctly without this margin due to the difference between placement of margin vs border.

Reply:
Could you add custom class instead? So we know its for demo purposes

Reply:
Done